frontend/tests/utils/helpers.ts

```diff
@@ -7,6 +7,9 @@ const { describe, skip } = test;
 
 export const TIMEOUT_MS = 3 * 1000;
 
+// (seve): We use TEST_ENV to describe the environment that playwright is running against. Sometimes the FE tests are run against a local instance of the app which points at a deployed instance of the backend.
+// We might want to rename some of these functions to better convey this.
+
```

Comment:
This function is removed in [this PR](https://github.com/chanzuckerberg/single-cell-data-portal/pull/3516)!